x/axelarcork/keeper/keeper.go

```diff
@@ -81,7 +81,7 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 // Scheduled Corks //
 /////////////////////
 
-func (k Keeper) SetScheduledCork(ctx sdk.Context, chainID uint64, blockHeight uint64, val sdk.ValAddress, cork types.Cork) []byte {
+func (k Keeper) SetScheduledCork(ctx sdk.Context, chainID uint64, blockHeight uint64, val sdk.ValAddress, cork types.AxelarCork) []byte {
```

Comment:
Same comment re: specificity of function names here. It just makes searching the codebase easier if they are differentiated.